requires.py

```diff
@@ -52,7 +54,9 @@ def aws_integration_ready():
         update_config_enable_aws()
     ```
     """
-    _instance_id_url = 'http://169.254.169.254/latest/meta-data/instance-id'
+    _metadata_url = 'http://instance-data/latest/meta-data/'
```

Comment:
what is http://instance-data?  a placeholder that gets substituted somewhere else?

Reply:
No, it's a DNS name provided by AWS.  It resolves to that IP (169.254.169.254).

The raw IP is explicitly given in [the docs](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html#instancedata-data-retrieval) so it might be better to stick with that, but the pseudo-hostname seems nicer to me.

Reply:
Hrm.  From https://forums.aws.amazon.com/message.jspa?messageID=536813 it sounds like AWS definitely does not recommend relying on it.  I'll revert it in the PR.